cli/tsc/99_main_compiler.js

```diff
@@ -1484,77 +1427,11 @@ delete Object.prototype.__proto__;
     };
   }
 
-  function transpile({
```

Comment:
Again, I would do it as a seperate PR...

Reply:
Opened https://github.com/denoland/deno/pull/7277, I'll rebase this branch after landing.